src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2009,6 +2009,15 @@ NetworkOPsImp::pubValidation(std::shared_ptr<STValidation> const& val)
         jvObj[jss::signing_time] = *(*val)[~sfSigningTime];
         jvObj[jss::data] = strHex(val->getSerializer().slice());
 
+        if (auto version = (*val)[~sfServerVersion]; version)
+            jvObj[jss::server_version] = std::to_string(*version);
+
+        if (auto cookie = (*val)[~sfCookie]; cookie)
+            jvObj[jss::cookie] = std::to_string(*cookie);
+
+        if (auto hash = (*val)[~sfValidatedHash]; hash)
+            jvObj[jss::validated_hash] = strHex(*hash);
+
```

Comment:
The second check in all of these are redundant. In other words,
```
if(auto x = X(); x)
  foo;
```
can be written as:
```
if(auto x = X())
  foo;
```

Reply:
Thanks @ximinez, I believe I've addressed your comment with the most recent commit.